admin.php

```diff
@@ -91,7 +117,37 @@ protected function html_table() {
             echo '<td>' . hsc($taggers) . '</td>';
             echo '</tr>';
         }
-
+        echo '<tr>';
+        echo '<td colspan="5" class="centeralign">';
+        echo '<span class="medialeft">';
+        echo '<button type="submit" name="fn[delete]" id="tagging__del">'.$this->getLang('admin delete_selected').'</button>';
+        echo '</tr>';
+        echo '</form>';
         echo '</table>';
+        
```

Comment:
Might be a good idea to transform this form into a dokuwiki\Form\Form as well, to make the code more consistent.